app/src/main/java/org/p2p/wallet/sdk/facade/SolendSdkFacade.kt

```diff
@@ -0,0 +1,211 @@
+        withContext(dispatchers.io) {
+            Timber.tag(TAG).i("Method createSolendDepositTransactions called")
+
+            val result = solendSdk.createSolendDepositTransactions(
+                solana_rpc_url = currentNetworkEnvironment.endpoint,
+                relay_program_id = relayProgramId,
+                amount = depositAmount,
+                symbol = currencySymbol,
+                ownerAddres = ownerAddress.value,
+                environment = solendEnvironment.sdkValue,
+                lendng_market_address = lendingMarketAddress.orEmpty(),
+                blockhash = currentBlockhash,
+                free_transactions_count = remainingFreeTransactionsCount,
+                need_to_use_relay = payFeeWithRelay,
+                pay_fee_in_token = feePayerToken?.let { gson.toJson(it) }.orEmpty(),
+                fee_payer_address = realFeePayerAddress.value
+            )
+            handleSdkResultWithHandler<SolendDepositTransactions>(result)
+        }
+    }
+
+    suspend fun createWithdrawTransactions(
+        relayProgramId: String,
+        depositAmount: ULong,
+        currencySymbol: String,
+        ownerAddress: Base58String,
+        lendingMarketAddress: String?,
+        currentBlockhash: String,
+        remainingFreeTransactionsCount: UInt,
+        payFeeWithRelay: Boolean,
+        feePayerToken: SolendFeePayerTokenData?,
+        realFeePayerAddress: Base58String
+    ) {
+        withContext(dispatchers.io) {
+            Timber.tag(TAG).i("Method createSolendWithdrawTransactions called")
+
+            val result = solendSdk.createSolendWithdrawTransactions(
+                solana_rpc_url = currentNetworkEnvironment.endpoint,
+                relay_program_id = relayProgramId,
+                amount = depositAmount,
+                symbol = currencySymbol,
+                owner_address = ownerAddress.value,
+                environment = solendEnvironment.sdkValue,
+                lendng_market_address = lendingMarketAddress.orEmpty(),
+                blockhash = currentBlockhash,
+                free_transactions_count = remainingFreeTransactionsCount,
+                need_to_use_relay = payFeeWithRelay,
+                pay_fee_in_token = feePayerToken?.let { gson.toJson(it) }.orEmpty(),
+                fee_payer_address = realFeePayerAddress.value
+            )
+            handleSdkResultWithHandler<SolendWithdrawTransactions>(result)
+        }
+    }
+
+    suspend fun getSolendCollateralAccounts(
+        ownerAddress: Base58String
+    ): List<SolendCollateralAccount> = withContext(dispatchers.io) {
+        Timber.tag(TAG).i("Method getSolendCollateralAccounts called")
+
+        methodResultMapper.fromSdk<SolendCollateralAccountsList>(
+            solendSdk.getSolendCollateralAccounts(
+                rpc_url = currentNetworkEnvironment.endpoint,
+                owner = ownerAddress.value
+            )
+        )
+            .getOrThrow()
+            .accounts
```

Comment:
Let's make a value of response and then return@withContext the response.accounts - that's easy to read